dev/com.ibm.ws.messaging.comms.client/src/com/ibm/ws/sib/jfapchannel/impl/NettyConnectionReadCompletedCallback.java

```diff
@@ -440,7 +440,6 @@ else if (thisConnection.getHeartbeatInterval() == 0)
 									((NettyNetworkConnection)vc).setHearbeatInterval(0);
 									readCtx.read(1, this, false, IOReadRequestContext.NO_TIMEOUT);      // F184828
 								} catch (NettyException e) {
-									// TODO Auto-generated catch block
 									e.printStackTrace();
```

Comment:
Why are we printing the stack trace here? Should we be rethrowing or logging the exception instead? Let's discuss

Reply:
Calling `setHearbeatInterval` can throw a NettyException if the Channel is null. We can log the exception with a debug statement or we can throw a RuntimeExcpetion that would be caught here:  https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.messaging.comms.client/src/com/ibm/ws/sib/jfapchannel/impl/NettyConnectionReadCompletedCallback.java#L582

Reply:
Need to discuss with the JMS team which option they would like us to implement.